mesop/server/state_session_test.py

```diff
@@ -377,10 +377,10 @@ def __init__(self):
 
   def get(
     self,
-    field_paths: Iterable[str] = None,  # type: ignore
+    field_paths: Optional[Iterable[str]] = None,  # type: ignore
```

Comment:
Hmm, these were intentional ignores since I was matching the code for the Firebase API. So looks like RUF ignored the type ignore here. Is there a different way to add an ignore for RUF?

I guess in this case, the changes are probably ok. But then we can remove the `type: ignore`

Reply:
yup, you can do an ignore with ruff (see: https://github.com/astral-sh/ruff/discussions/3442) - I've ignored this specific error for the file